includes/reader-revenue/templates/myaccount-edit-account.php

```diff
@@ -67,10 +68,24 @@
 	<?php \do_action( 'woocommerce_edit_account_form_end' ); ?>
 </form>
 
+<hr />
+
+<h3><?php \esc_html_e( 'Reset Password', 'newspack' ); ?></h3>
+
 <p class="woocommerce-form-row woocommerce-form-row--wide form-row form-row-wide">
 	<a href="<?php echo '?' . \esc_attr( $newspack_reset_password_arg ) . '=' . \esc_attr( \wp_create_nonce( $newspack_reset_password_arg ) ); ?>">
 		<?php \esc_html_e( 'Email me a password reset link', 'newspack' ); ?>
 	</a>
 </p>
 
+<hr />
+
+<h3><?php \esc_html_e( 'Delete Account', 'newspack' ); ?></h3>
+
+<p class="woocommerce-form-row woocommerce-form-row--wide form-row form-row-wide">
+	<a href="<?php echo '?' . \esc_attr( $newspack_delete_account_arg ) . '=' . \esc_attr( \wp_create_nonce( $newspack_delete_account_arg ) ); ?>">
+		<?php \esc_html_e( 'Email me a link for account deletion', 'newspack' ); ?>
```

Comment:
Non-blocking suggestion:

```suggestion
		<?php \esc_html_e( 'Request account deletion', 'newspack' ); ?>
```